storybook/stubs/NimModules/GlobalUtils.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.14
```

Comment:
Generally now all stuff here under `NimModules` is not stubs but mocks. We shouldn't mix up those two concepts.

Reply:
Initially I thought about adding the modules in mocks folder, but then this short read convinced me that we're using stubs.
https://martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs

Reply:
Thanks for the article. Generally in our case is quite specific bc it's qml. Those stubs are mainly for import system to be found and recognized as a specific type. No implementation at all. Only mocs have interface expected by the components. I see it's high time to write readme for storybook clarifying those things :)